### DISCUSSION TITLE
[Prettier Plugin] Line breaks after sorting classes

### DISCUSSION THREAD

**Discussion by Edmund1645** [tailwindlabs/tailwindcss#10005] [Ideas]
I recently discovered the prettier plugin and it's been a delight to use! I wanted to go in prettier's customisations to make the classes wrap to a new line but it seems that functionality has been removed in [Prettier v2.5.0](https://prettier.io/blog/2021/11/25/2.5.0.html#collapse-html-class-attributes-onto-one-line-11827httpsgithubcomprettierprettierpull11827-by-jlongsterhttpsgithubcomjlongster).

It would be great to have this as part of the Prettier plugin for Tailwind.

**Comment by ony3000** [comment]
Hello! I am leaving a comment because it seems to be a similar topic to #7763.

This is the first time I discovered that a class name line break function existed in prettier in the past! :astonished:
There is a [prettier plugin](https://www.npmjs.com/package/prettier-plugin-classnames) that I released this year. Maybe this will help.
